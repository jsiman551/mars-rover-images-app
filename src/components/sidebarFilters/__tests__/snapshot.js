import { render } from '@testing-library/react';
import SidebarFilters from '../index';

it('renders siderbar filters unchanged', () => {
  const { container } = render(<SidebarFilters />);
  expect(container).toMatchSnapshot();
});
