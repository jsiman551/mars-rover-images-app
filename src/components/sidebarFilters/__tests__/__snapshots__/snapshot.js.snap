// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders siderbar filters unchanged 1`] = `
<div>
  <div
    class="css-0"
    data-testid="sidebar-container"
  />
</div>
`;
